--- gst-plugins-ugly-1.0.0/configure.ac.orig	2012-09-23 18:52:42.000000000 +0200
+++ gst-plugins-ugly-1.0.0/configure.ac	2012-10-05 17:48:00.575430221 +0200
@@ -351,7 +351,7 @@
 dnl *** x264 (MPEG-4 part 10/h.264/AVC encoder) ***
 translit(dnm, m, l) AM_CONDITIONAL(USE_X264, true)
 AG_GST_CHECK_FEATURE(X264, [x264 plug-in], x264, [
-  AG_GST_PKG_CHECK_MODULES(X264, x264 >= 0.120.0)
+  AG_GST_PKG_CHECK_MODULES(X264, x264 >= 0.120)
 ])
 
 else
